spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/batch/BatchAutoConfigurationTests.java

```diff
@@ -233,7 +233,7 @@ void testRenamePrefix() {
 				.withUserConfiguration(TestConfiguration.class, EmbeddedDataSourceConfiguration.class,
 						HibernateJpaAutoConfiguration.class)
 				.withPropertyValues("spring.datasource.generate-unique-name=true",
-						"spring.batch.jdbc.schema:classpath:batch/custom-schema-hsql.sql",
+						"spring.batch.jdbc.schema:classpath:batch/custom-schema-h2.sql",
```

Comment:
Can you share why you're switching the test to use H2 instead of HSQL?  If the purpose is to test that Spring Batch works with their H2 scripts, that's something that should be done as part of a Spring Batch release.

Reply:
The scope of the test is that the property `spring.batch.jdbc.schema` works as expected. It doesn't actually matter which DB is used.

I changed the name because the test does not use HSQL but H2. I noticed that as it fails with upgraded H2 which it shouldn't if HSQL were used.

As the test is not really about the concrete DB and the new DDL works with both HSQL and H@, would it be okay to rename the script to `custom-schema.sql`?